gitclone/git.go

```diff
@@ -107,7 +111,18 @@ func getDiffFile(buildURL, apiToken string, prID int) (string, error) {
 
 func run(c *command.Model) error {
 	log.Infof(c.PrintableCommandArgs())
-	return c.SetStdout(os.Stdout).SetStderr(os.Stderr).Run()
+	var buffer bytes.Buffer
+	outWriter := io.MultiWriter(os.Stdout, &buffer)
+
+	err := c.SetStdout(outWriter).SetStderr(outWriter).Run()
+	if err != nil {
+		if errorutil.IsExitStatusError(err) {
+			return errors.New(buffer.String())
+		}
+		return fmt.Errorf("command execution failed, %s", err)
```

Comment:
i do not see the added value of `command execution failed` prefix, previously we just returned and logged the error

Reply:
True, this only happens if the error is not exit status error tough.